app/controllers/clients_controller.rb

```diff
@@ -0,0 +1,53 @@
+class ClientsController < ApplicationController
+    before_action :set_client, only: [:show, :update, :destroy]
```

Comment:
similar to https://github.com/lukasz-ligizynski/Project/pull/2#discussion_r639644959